### DISCUSSION TITLE
Sudden high memory usage and growth

### DISCUSSION THREAD

**Discussion by Tony-MF** [plausible/analytics#2682] [Self-Hosted Support]
Hi,

This started to happen recently. Is this an expected behaviour?

<img width="647" alt="Screenshot 2023-02-15 at 18 03 47" src="https://user-images.githubusercontent.com/73653258/219100462-c7371b1a-3b2c-496f-904f-0b46eedfa279.png">

Here is docker stats:
```
CONTAINER ID   NAME                            CPU %     MEM USAGE / LIMIT     MEM %     NET I/O           BLOCK I/O        PIDS
174869e95716   hosting_plausible_1             487.52%   38.76GiB / 62.81GiB   61.72%    54.4GB / 58.8GB   99.1MB / 4.1kB   105
8c50ee48de85   hosting_plausible_db_1          0.40%     73.29MiB / 62.81GiB   0.11%     136MB / 73.6MB    86.9MB / 300MB   18
73cdab340503   hosting_mail_1                  0.00%     15.07MiB / 62.81GiB   0.02%     163kB / 84.1kB    87.6MB / 266kB   1
391894a2eada   hosting_plausible_events_db_1   2.07%     298.6MiB / 62.81GiB   0.46%     30.9GB / 671MB    18GB / 2.42MB    290
```

The drop in memory usage happens when I restart all containers. I'm afraid of what will happen when I run out of memory :)

**Comment by ukutaht** [comment]
What version are you on? I'm asking because the only known memory leak was fixed in `v1.5.0`

**Reply by Tony-MF** [reply]
The docker tag of plausible/analytics is "latest".
